From 013ebd1cba185b56521b257fa99f2bca333ebde6 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:47:18 -0400
Subject: [pci] hotplug/cpqphp: Fix hex vs decimal typo in cpqhpc_probe()

Message-id: <20140806034717.1438.94522.stgit@gir.stowe>
Patchwork-id: 87439
O-Subject: [RHEL7.1 PATCH 30/69] PCI: cpqphp: Fix hex vs decimal typo in cpqhpc_probe()
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: 357fe8576374652be3dfe1c829c3c0a4dd40c24f

commit 357fe8576374652be3dfe1c829c3c0a4dd40c24f
Author: Dan Carpenter <dan.carpenter@oracle.com>
Date:   Mon Feb 10 18:23:50 2014 +0300

    PCI: cpqphp: Fix hex vs decimal typo in cpqhpc_probe()

    This is a static checker fix and I can't test it, but from the context it
    definitely looks like hexadecimal 0x20 was intended here instead of decimal
    20.

    Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/hotplug/cpqphp_core.c b/drivers/pci/hotplug/cpqphp_core.c
index 31273e1..037e261 100644
--- a/drivers/pci/hotplug/cpqphp_core.c
+++ b/drivers/pci/hotplug/cpqphp_core.c
@@ -920,12 +920,12 @@ static int cpqhpc_probe(struct pci_dev *pdev, const struct pci_device_id *ent)
     bus->max_bus_speed = PCI_SPEED_100MHz_PCIX;
     break;
    }
-   if (bus_cap & 20) {
+   if (bus_cap & 0x20) {
     dbg("bus max supports 66MHz PCI-X\n");
     bus->max_bus_speed = PCI_SPEED_66MHz_PCIX;
     break;
    }
-   if (bus_cap & 10) {
+   if (bus_cap & 0x10) {
     dbg("bus max supports 66MHz PCI\n");
     bus->max_bus_speed = PCI_SPEED_66MHz;
     break;
-- 
1.7.1